tools/converter/reports_converter.cpp

```diff
@@ -0,0 +1,126 @@
+#include <cstring>
+#include <fstream>
+#include <iostream>
+#include <map>
+#include <vector>
+
+#ifdef SONATA_REPORT_HAVE_MPI
+#include <mpi.h>
+#endif
+
+#include "binary_reader/binary_reader.h"
+#include <bbp/sonata/reports.h>
+#include <utils/logger.h>
+
+using namespace bbp::binary_reader;
+
+static void show_usage(std::string name) {
```

Comment:
(I think there are such nitpicks at couple of places like this but I wouldn't make big deal of those here)

```suggestion
static void show_usage(const std::string& name) {
```